### ISSUE TITLE
DateTimePicker style error

### ISSUE THREAD

**Issue by tcqq** [Gericop/DateTimePicker#6]
When theme Uses `MaterialComponents` Instead of `AppCompat` the Button style error.

![tim 20180618222100](https://user-images.githubusercontent.com/10644057/41541848-023c7f30-7346-11e8-91cc-d5a828c31658.png)![tim 20180618222103](https://user-images.githubusercontent.com/10644057/41541856-037fc4f6-7346-11e8-9cfb-a6ba2db83f82.png)

Style:
-
```
    <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorSecondary">@color/colorSecondary</item>
    </style>
```

**Comment by tcqq** [comment]
Fixed. #7

**Comment by gregkorossy** [comment]
Fixed in v**1.0.7-alpha3** (technically in alpha1 but that one didn't produce this error).
